cdk/lib/admin-console.ts

```diff
@@ -140,6 +140,7 @@ export class AdminConsole extends GuStack {
         name: 'timestamp',
         type: AttributeType.STRING,
       },
+      timeToLiveAttribute: 'ttlInSecondsSinceEpoch',
```

Comment:
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html

Reply:
Thanks for sharing the resource!